Gem/Code/Source/Spawners/RoundRobinSpawner.cpp

```diff
@@ -30,6 +30,12 @@ namespace MultiplayerSample
             return AZStd::make_pair<Multiplayer::PrefabEntityId, AZ::Transform>(Multiplayer::PrefabEntityId(), AZ::Transform::CreateIdentity());
         }
 
+        if (m_spawnIndex >= m_spawners.size())
+        {
+            AZLOG_WARN("RoundRobinSpawner has an out-of-bounds spawner index. Resetting spawn index to 0.")
```

Comment:
Would question if this a warning ie we fixed it and do not appear to expect user to take action. Is it worth having log message here?

Reply:
This will never get hit if ppl use our SpawnerComponent. It could happen if some other spawner the user wrote didn't unregister itself. I'd like users to know if it happens, because if it does something is off with their logic.